fastlane/lib/fastlane/actions/mailgun.rb

```diff
@@ -86,7 +86,12 @@ def self.available_options
                                       env_name: "MAILGUN_REPLY_TO",
                                       description: "Mail Reply to",
                                       optional: true,
-                                      is_string: true)
+                                      is_string: true),
+          FastlaneCore::ConfigItem.new(key: :attachment,
+                                      env_name: "MAILGUN_ATTACHMENT",
+                                      description: "Mail Attachments",
```

Comment:
Could you update the description to describe better on what to pass? An array? One item as string? The data or the file path(s)?